mods/d2k/rules/starport.yaml

```diff
@@ -21,6 +21,7 @@ trike.starport:
 	Inherits: trike
 	Buildable:
 		Queue: Starport
+		Prerequisites: starport
```

Comment:
This isn't necessary. Only the starport provides the starport queue, so adding it as a prerequisite is redundant.